tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+---
+
+- name: Check if {{ prometheus_root_dir }} directory exists
+  stat:
+    path: "{{ prometheus_root_dir }}"
+  register: directory_exists
+
+- import_tasks: configure-directories.yml
+  when: directory_exists.stat.exists == false
+
+- name: Download Prometheus Kafka exporter
+  become: yes
+  get_url:
+    url: "{{ kafka_exporter_tar_gz_url }}"
+    dest: "{{ prometheus_dist_dir }}"
+    validate_certs: no
+
+- name: Extract {{ kafka_exporter_dir_path }}.tar.gz into {{ prometheus_dist_dir }}
+  become: yes
+  unarchive:
+    src: "{{ prometheus_dist_dir }}/{{ kafka_exporter_dir_path }}.tar.gz"
+    dest: "{{ prometheus_dist_dir }}"
+    remote_src: yes
+
+- name: Delete "{{ prometheus_dist_dir }}/{{ kafka_exporter_dir_path }}.tar.gz"
+  become: yes
+  file:
+    path: "{{ prometheus_dist_dir }}/{{ kafka_exporter_dir_path }}.tar.gz"
+    state: absent
+
+- name: Symlink to Prometheus Kafka exporter binary
+  become: yes
+  file:
+    src: "{{ prometheus_dist_dir }}/{{ kafka_exporter_dir_path }}/kafka_exporter"
+    dest: "{{ prometheus_bin_dir }}/kafka_exporter"
+    state: link
+
+- name: Create Prometheus Kafka exporter systemd service file
+  become: yes
+  template:
+    src: templates/{{ service_name }}.service.j2
+    dest: "{{ kafka_exporter_systemd_dir }}/{{ service_name }}.service"
+
+- name: Enable {{ service_name }} service
+  become: yes
+  systemd:
+    daemon_reload: yes
+    name: "{{ service_name }}"
+    enabled: yes
```

Comment:
I would make this configurable as we'll need to disable this for AMIs out of the box. It would probably be worth renaming the task to `Configure` too